### PR TITLE
Supplementary testcase for py module 

### PR DESCRIPTION
**Pull request by zeshuai007**
- [ ] Did you create an [Apache Jira](https://issues.apache.org/jira/projects/THRIFT/issues/) ticket?  (not required for trivial changes)
- [ ] If a ticket exists: Does your pull request title follow the pattern "THRIFT-NNNN: describe my issue"?
- [ ] Did you squash your changes to a single commit?  (not required, but preferred)
- [ ] Did you do your best to avoid breaking changes?  If one was needed, did you label the Jira ticket with "Breaking-Change"?
- [ ] If your change does not involve any code, add ` [skip ci]` at the end of your pull request to free up build resources.

### CODE REVIEW

**Comment by dcelasun** [comment]
This broke [CI](https://api.travis-ci.org/v3/job/672411817/log.txt) due to style issues.

Please ping me after fixing and I'll reopen.

**Comment by zeshuai007** [comment]
hi, @dcelasun 
I have modified it locally and can resubmit this PR.  can you reopen this PR?

File: lib/py/test/thrift_transport.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import unittest
```
Comment (Jens-G) [inline lib/py/test/thrift_transport.py:1]: ASF header missing

**Comment by zeshuai007** [comment]
@Jens-G
 I have fixed the conflict, and the CI has passed.

**Comment by dcelasun** [comment]
LGTM.

**Pull request merged**